docs/networking/networking.md

```diff
@@ -57,13 +57,13 @@ Create a [Service of type LoadBalancer](https://kubernetes.io/docs/concepts/serv
 
 ### Controlling ServiceLB Node Selection
 
-To exclude nodes from being used by ServiceLB, add the following label to the nodes that should host ServiceLB Pods. All unlabeled nodes will not be used for ServiceLB.
+In order for ServiceLB to assign nodes to pools, nodes must have the following label:
```

Comment:
We still need the prior language explaining that adding the `enablelb` annotation to any node disables it on any unlabeled nodes. We just need to add a bit about `lbpool` also requiring `enablelb` on any pooled nodes.